src/azure-cli/azure/cli/command_modules/databoxedge/generated/_params.py

```diff
@@ -0,0 +1,253 @@
+        c.argument('device_name', options_list=['--name', '-n', '--device-name'], type=str, help='The device name.',
+                   id_part='name')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('databoxedge device show-update-summary') as c:
+        c.argument('device_name', options_list=['--name', '-n', '--device-name'], type=str, help='The device name.',
+                   id_part='name')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('databoxedge device wait') as c:
+        c.argument('device_name', options_list=['--name', '-n', '--device-name'], type=str, help='The device name.',
+                   id_part='name')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('databoxedge alert list') as c:
+        c.argument('device_name', options_list=['--device-name', '-d'], type=str, help='The device name.')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('databoxedge alert show') as c:
+        c.argument('device_name', options_list=['--device-name', '-d'], type=str, help='The device name.',
+                   id_part='name')
+        c.argument('name', options_list=['--name', '-n'], type=str, help='The alert name.', id_part='child_name_1')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('databoxedge bandwidth-schedule list') as c:
+        c.argument('device_name', options_list=['--device-name', '-d'], type=str, help='The device name.')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('databoxedge bandwidth-schedule show') as c:
+        c.argument('device_name', options_list=['--device-name', '-d'], type=str, help='The device name.',
+                   id_part='name')
+        c.argument('name', options_list=['--name', '-n'], type=str, help='The bandwidth schedule name.',
+                   id_part='child_name_1')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('databoxedge bandwidth-schedule create') as c:
+        c.argument('device_name', options_list=['--device-name', '-d'], type=str, help='The device name.')
+        c.argument('name', options_list=['--name', '-n'], type=str, help='The bandwidth schedule name which needs to '
+                   'be added/updated.')
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('start', type=str, help='The start time of the schedule in UTC.')
+        c.argument('stop', type=str, help='The stop time of the schedule in UTC.')
+        c.argument('rate_in_mbps', type=int, help='The bandwidth rate in Mbps.')
+        c.argument('days', nargs='+', help='The days of the week when this schedule is applicable.')
```

Comment:
why the days parameter is list array?

Reply:
Users can create same bandwidth rule for several days, for example, 10mbps between 09:00:00 and 18:00:00 for both 'Saturday' and 'Sunday'

Reply:
Do we have test to cover such scenario?

Reply:
To test bandwidth-schedule, the device need to be registered which need a vm prepared and some manual work on web page. So it's hard to write tests.